src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -153,6 +155,34 @@ Qgs3DMapCanvasWidget::Qgs3DMapCanvasWidget( const QString &name, bool isDocked )
   } );
   mOptionsMenu->addAction( mActionEnableEyeDome );
 
+  mActionSync2DNavTo3D = new QAction( tr( "Sync 2D Navigation to 3D Navigation" ) );
+  mActionSync2DNavTo3D->setCheckable( true );
+  connect( mActionSync2DNavTo3D, &QAction::triggered, this, [ = ]( bool enabled )
+  {
+    Qgis::ViewSyncModeFlags syncMode = mCanvas->map()->viewSyncMode();
+    syncMode.setFlag( Qgis::ViewSyncModeFlag::Sync2DTo3D, enabled );
+    mCanvas->map()->setViewSyncMode( syncMode );
+  } );
+  mOptionsMenu->addAction( mActionSync2DNavTo3D );
+
+  mActionSync3DNavTo2D = new QAction( tr( "Sync 2D Navigation to 3D Navigation" ) );
+  mActionSync3DNavTo2D->setCheckable( true );
+  connect( mActionSync3DNavTo2D, &QAction::triggered, this, [ = ]( bool enabled )
+  {
+    Qgis::ViewSyncModeFlags syncMode = mCanvas->map()->viewSyncMode();
+    syncMode.setFlag( Qgis::ViewSyncModeFlag::Sync3DTo2D, enabled );
+    mCanvas->map()->setViewSyncMode( syncMode );
+  } );
+  mOptionsMenu->addAction( mActionSync3DNavTo2D );
+
+  mShowFrustumPolyogon = new QAction( tr( "Show 3D view frustum in main map canvas" ) );
```

Comment:
```suggestion
  mShowFrustumPolyogon = new QAction( tr( "Show Visible Camera Area in 2D Map View" ), this );
```

Reply:
("view frustum" is a very technical term -- I doubt most users would understand that)